docs/data-ai/capture-data/_index.md

```diff
@@ -2,10 +2,117 @@
 linkTitle: "Capture data"
 title: "Capture data"
 weight: 100
-layout: "empty"
+layout: "docs"
 type: "docs"
-empty_node: true
```

Comment:
I think we do actually want to keep these top level pages empty. There's been a lot of confusion on whether people get confused with some of these having content and others not having content. I think in this case (1) we have a child page with almost the same name so the content should be merged onto that page or this page's content needs to go into reference. (2) with the current design the top level pages really don't feel like they should have content themselves. No other top level page currently does. (3) this isn't the first thing you'd want to tell someone. This is the info for people who want to know more so it should come after they get the basics.